### DISCUSSION TITLE
Product page not working properly

### DISCUSSION THREAD

**Discussion by jboehmecw** [bootscore/bootscore#440] [Q&A]
I have tried everything I can think of, but I cannot get the product page to work. It won't let me add a product. Every other page after this page works as expected.

It should work like this: https://www.traxsideskating.com/wp/product/blackout-party-package/

It looks like this: https://woocommerce-594904-3301961.cloudwaysapps.com/product/blackout-party-package

I tried turning off javascript, styling, updating files, etc. I tried a different theme and it did work (if you look I have something in the cart from my trying things out). 

It appears to me javascript files are conflicting. That didn't seem to help when I would turn one off at a time. Same with styling. I don't see any conflicts in the Google console.

 I've also tried turning off the plugins one at a time. That didn't work either. 

I am using the child theme. 

Any suggestions would be tremendously appreciated!

**Comment by crftwrk** [comment]
It seems there is a conflict in theme/your child and or a plugin. Do following steps:

- Download and save your existing child
- Install and activate a fresh bootcommerce-child https://github.com/bootscore/bootcommerce-child/releases/download/v5.2.3.1/bootcommerce-child-main.zip
- Disable ALL plugins except WooCommerce

You should now have a simple product page exactly like on bootscore.me. If so, do this:

- Enable the calendar plugin

Check product page again. If it still not work, disable AJAX cart in bootcommerce-child's `functions.php`, and disable AJAX in WooCommerce as described here https://bootscore.me/documentation/woocommerce/#Disable_ajax_cart. 

Usually it should work now, because it seems that calendar plugin doesn't like it. If it still not work, let me know and share a link to the calendar plugin site.

- If it works, enable each plugin step-by-step and check product page on each change you made.
- Install your child and disable AJAX as described above. If it works, good.
- If it not works, go back to the fresh bootcommerce-child and transfer changes step-by-step from your child to the new one. Check product page on each change again.

Hope that helps

**Reply by jboehmecw** [reply]
Thank you. I appreciate it!  I'll give it a shot!

**Reply by jboehmecw** [reply]
I performed what you said. It didn't work. However, it did get me thinking about things.

I realized this morning I accidentally removed some js files that were needed.

I **VERY** much appreciate you answering me! It did help!!!

**Reply by crftwrk** [reply]
You're welcome 👍
